Update Digital Assets guides (use LSP20) (#481)
lukso-network/docs: docs/guides/digital-assets/transfer-lsp7-digital-asset.md

Reviewer: it should be "transfer **a** LSP7 ..." in the sidebar

<img width="296" alt="image" src="https://user-images.githubusercontent.com/31145285/232038853-e1c464f4-c995-4614-a3f7-69c9c50a6541.png">

### Step 1 - Setup imports and constants

> At this point you will need a private key, this EOA should be a controller address for the UP with proper [**permissions**](../../standards/universal-profile/lsp6-key-manager.md#permissions) in order to transfer some tokens as well as the `LSP7Mintable` _token contract address_ and the _address of the Universal Profile_ that has some tokens.

Reviewer: Break it down in bullet points.

-----
at this point, you will need:

- the address + ABI of the LSP7DigitalAsset token contract
- the address + ABI of the Universal Profile
- the private key of the controller of your Universal Profile

Let's also not mention here about permission, but just put a link to the LSP6 page to explain what is a controller.

</Tabs>

### Step 3 - Setup the token trasnfer calldata

> Now we need to prepare the calldata that we will use in order to transfer tokens from a Universal Profile to another.s

Reviewer: ```suggestion
Now we need to prepare the calldata to transfer tokens from a Universal Profile to another UP.
```
